### PR TITLE
Added Spanish translation of the Add Contributions! CTA

### PR DESCRIPTION
**Pull request by Chinonyemildred**
In response to issue #2835, I created a pull request to create a precise Call-to-action which invites users to make contributions to the tutorial.

In this PR, I added the Spanish translation of the **Add Contributions!** CTA to the metadata es.yml file. 

Is this okay?

### CODE REVIEW

**Comment by Chinonyemildred** [comment]
#2835

File: metadata/lang/es.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ questions: "Preguntas"
 objectives: "Objetivos"
 requirements: "Requisitos"
 authors: "Autores"
+add contributions: "¡Agregue contribuciones!"
```
Comment (shiltemann) [inline metadata/lang/es.yml:7]: I am not sure if spaces will work here, can you change it to `add-contributions` in both files?

**Review by shiltemann — COMMENTED** [review]
Perfect, thanks @Chinonyemildred!

File: metadata/lang/es.yml (hunk 1 above)
Comment (Chinonyemildred) [reply metadata/lang/es.yml:7]: Okay, I'll do that. Thank you❤

**Comment by Chinonyemildred** [comment]
Please how do I resolve the yml file failed test?

**Review by shiltemann — APPROVED** [review]

**Comment by shiltemann** [comment]
Thanks a lot @Chinonyemildred!

**Comment by Chinonyemildred** [comment]
Amazing...thank you @shilteman!

**Pull request merged**